nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -203,7 +203,7 @@ def __init__(self, manifests_files: Union[str, List[str]], *args, **kwargs):
         """
 
         ids, audio_files, durations, texts, offsets, speakers, orig_srs, langs = [], [], [], [], [], [], [], []
-        for item in manifest.item_iter(manifests_files):
+        for item in manifest.item_iter(manifests_files, data_prefix=data_prefix):
```

Comment:
Changing this signature requires changes in all data loaders and classes that call this signature. Try to revert or redesign this part